hrui/hrui.php

```diff
@@ -588,9 +588,17 @@ function hrui_civicrm_managed(&$entities) {
   return _hrui_civix_civicrm_managed($entities);
 }
 
+/**
+ * Implements hook_civicrm_navigationMenu().
+ *
+ * @param Array $params List of menu items
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_navigationMenu
+ */
 function hrui_civicrm_navigationMenu(&$params) {
   _hrui_customImportMenuItems($params);
   _hrui_coreMenuChanges($params);
+  _createHelpMenu($params);
```

Comment:
I didn't mention it until now, but it might be nice to follow the convention here and use `_hrui_` as a prefix for these function names.